packages/stunner/PKGBUILD

```
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stunner
+pkgver=0.5.8
+pkgrel=1
+pkgdesc='Test and exploit STUN, TURN and TURN over TCP servers.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-networking' 'blackarch-webapp')
+url='https://github.com/firefart/stunner'
+license=('CC BY-NC-SA 4.0')
```

```suggestion
license=('CC-BY-NC-SA-4.0')
```

see codenames at https://spdx.org/licenses/